spec/acceptance/class_spec.rb

```diff
@@ -6,9 +6,17 @@
   it_behaves_like 'an idempotent resource' do
     let(:manifest) do
       <<-PUPPET
-      # FIXME: https://github.com/puppetlabs/puppetlabs-apt/pull/1015
-      ensure_packages('apt-transport-https')
-      Package['apt-transport-https'] -> Class['apt::update']
+      if fact('os.family') == 'debian' {
+        # FIXME: https://github.com/puppetlabs/puppetlabs-apt/pull/1015
+        ensure_packages('apt-transport-https')
+        Package['apt-transport-https'] -> Class['apt::update']
+      }
+
+      if fact('os.family') == 'redhat' {
```

Comment:
Shouldn't this be `== 'RedHat'` ?
Same comment for L9

Reply:
:+1:  for consistency with the rest of the module.  String comparisons are case insensitive for ASCC chars with puppet:
https://puppet.com/docs/puppet/7/lang_expressions.html#lang_exp_comparison_operators

Reply:
Fixed